Commands/Commands.cs

```diff
@@ -23,14 +24,21 @@ public async Task GetLatency(CommandContext ctx)
                 .Build();
             await ctx.Channel.SendMessageAsync(embed: latency);
         }
-        [Command("F")]
-        private async Task F()
-        {
-            await ReplyAsync("F");
-        }
-        [Command("say")]
-        [Summary("say what user gave as input")]
-        public Task SayAsync([Remainder] [Summary("What shoukd the bot say")] string echo)
-            => ReplyAsync(echo);
+
+        //F command
+        [Command("F"), Aliases("f")]
+        [RequiresRole("Member")]
+        [Cooldown(3, 120, CooldownBucketType.User)]
+        [Description("Returns F")]
+        public async Task SayF()
+            => await ReplyAsync("F");
+
+        //Echo command
+        [Command("say"), Aliases("Say")]
+        [RequiresRole("Member")]
+        [Cooldown(3, 120, CooldownBucketType.User)]
```

Comment:
We don't need a cooldown, the member role was added as it is considered that members would not spam that. We do not need it.

Reply:
But you said we need a cooldown